### ISSUE TITLE
Kanban Board View

### ISSUE THREAD

**Issue by nakul80** [frappe/erpnext#44451]
Kanban Board should not dynamically adjust the width too much. Due to a custom field being added to the view, a single column is widening to two times the length of a regular kanban column. A simple wrap to the next line would be better visually.

Also, hardly any options for customizing.

**Comment by johnwslongland** [comment]
I would agree with adding some more customizing options. I am trying to fix the sorting order within a column and cannot find a way ?

**Comment by VadymKhvoinytskyi** [comment]
Same here, long text field messes up view completely, extending to two screens width. It would be cool be able to adjust it in kanban settings or something like print width in doctype customization
